domain/consensus/utils/hashserialization/utxo.go

```diff
@@ -49,25 +85,21 @@ func serializeOutpoint(w io.Writer, outpoint *externalapi.DomainOutpoint) error
 }
 
 func serializeUTXOEntry(w io.Writer, entry *externalapi.UTXOEntry) error {
-	buf := [8 + 1 + 8]byte{}
+	buf := [8 + 1 + 8 + 8]byte{}
 	// Encode the blueScore.
 	binary.LittleEndian.PutUint64(buf[:8], entry.BlockBlueScore)
 
 	buf[8] = serializeUTXOEntryFlags(entry)
 
 	binary.LittleEndian.PutUint64(buf[9:], entry.Amount)
 
+	binary.LittleEndian.PutUint64(buf[17:], uint64(len(entry.ScriptPublicKey)))
```

Comment:
This function has seen been changed in @svarogg's branch. Please make sure to align it with how he has done it.

Reply:
It's aligned. it's just weird that this was the only place in the file that used `WriteElement` so I replaced it to be like the rest (which were also done for performance reasons)